corehq/apps/app_manager/const.py

```diff
@@ -22,6 +22,7 @@
 USERCASE_TYPE = 'commcare-user'
 USERCASE_ID = 'usercase_id'
 USERCASE_PREFIX = 'user/'
+WEBUSER_USERCASE_TYPE = 'web-user'
```

Comment:
Where does this come from?  Looking at the usercase code, it looks like web users would use the same case type as mobile workers.  I don't see any references to the string `"web-user"` that look relevant.

Reply:
Yeah this was probably unnecessary. I had put this in anticipation of using this string in more places but I only used it here:
`CommCareCase.objects.get_case_by_external_id(domain, self._id, 'web-user')`

I went and ahead and got rid of it.

Reply:
I'm asking more about the use of `'web-user'` as the case type.  Won't web user's usercases be of the type `'commcare-user'`, just like mobile workers?  It's kinda hard to track down where that's configured, but I don't see anything that looks like it uses a different case type for web users.

Reply:
I went ahead and looked at this [PR](https://github.com/dimagi/commcare-hq/pull/30776) where this is configured and you're right it's `commcare-user`. I've went ahead and fixed this.